src/transformers/models/dpt/modeling_dpt.py

```diff
@@ -1046,9 +1058,17 @@ def __init__(self, config):
             self.projection = nn.Conv2d(256, 256, kernel_size=(3, 3), stride=(1, 1), padding=(1, 1))
 
         features = config.fusion_hidden_size
+
+        image_size = (
+            config.backbone_config.image_size
```

Comment:
Let's not rely on the backbone config being present. The config being there or not defines how the backbone is loaded, but we shouldn't use it to parameterise the model itself